### ISSUE TITLE
safInvariant not working from Python (template uninstantiated)

### ISSUE THREAD

**Issue by videlec** [flatsurf/intervalxt#91]
```
from pyintervalxt import intervalxt 
p = [1, 0]
lengths = [2, 1]                                
lengths = intervalxt.sample.Lengths[int](lengths)   
iet = intervalxt.makeIET(lengths, p)
iet.safInvariant()
```
gives
```
input_line_61:6:133: error: implicit instantiation of undefined template 'std::valarray<__gmp_expr<mpq_t, mpq_t> >'
  ...>) (((const intervalxt::IntervalExchangeTransformation*)obj)->safInvariant());
                                                                   ^
/usr/lib/gcc/x86_64-pc-linux-gnu/9.2.1/../../../../include/c++/9.2.1/bits/range_access.h:102:32: note: template is declared here
  template<typename _Tp> class valarray;
                               ^
input_line_61:10:65: error: implicit instantiation of undefined template 'std::valarray<__gmp_expr<mpq_t, mpq_t> >'
      ((const intervalxt::IntervalExchangeTransformation*)obj)->safInvariant();
                                                                ^
/usr/lib/gcc/x86_64-pc-linux-gnu/9.2.1/../../../../include/c++/9.2.1/bits/range_access.h:102:32: note: template is declared here
  template<typename _Tp> class valarray;
                               ^
```

**Comment by saraedum** [comment]
Fixed by adding `valarray` to `cppyy.hpp`.

**Comment by videlec** [comment]
fixed via #93
